content/en/docs/reference/access-authn-authz/webhook.md

```diff
@@ -29,6 +29,8 @@ file format. Within the file "users" refers to the API Server webhook and
 A configuration example which uses HTTPS client auth:
 
 ```yaml
+#Kubernetes API version
+apiVersion: v1
```

Comment:
Can you help point me to the issue/pr/source/... where the 'apiVersion' field becomes required? The YAML is a kubeconfig struct where apiVersion is not strictly required IIRC.

Reply:
I have seen an intermittent issues restarting api-server (fails to restart) after configuring web-hook. But it work flawlessly when api version is mentioned.
Not sure if there is an existing issue for this.

Reply:
ping @deads2k, @lavalamp, PTAL. Thanks.

Reply:
You should always specify it IMO. We make assumptions given the context it's used in but better to specify it.

apiVersion includes the group, if we want to be technical.

You should also specify the kind.

These lines don't merit a comment and if you keep the comment anyway, place a space after the #.

Reply:
@lavalamp I fixed comment. Although I had no luck with grop and kind. For Authentication I tried
apiVersion: authentication.k8s.io/v1
kind: TokenReview

But with this api-server fails to restart. Am I missing anything here?